crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -929,6 +916,29 @@ impl HolochainP2pActor {
             .collect::<Vec<_>>())
     }
 
+    /// Randomly selects [`PARALLEL_GET_AGENTS_COUNT`] peers containing the op at the given location.
```

Comment:
```suggestion
    /// Randomly selects [`PARALLEL_GET_AGENTS_COUNT`] agents whose storage arcs contain the given location and returns the agents with a peer URL for each.
```

Something like that? This is selecting agents technically and it's not an "op" exactly, there may be many ops with unique hashes, at the same location.

Reply:
Sure, I typically try to keep the first line of a doc-comment short which is why I had such an abridged version.

Reply:
Done: ca089d1ce9844b1d94008c3236f73d415f398481